realsense2_camera/CMakeLists.txt

```diff
@@ -115,6 +115,7 @@ find_package(nav_msgs REQUIRED)
 find_package(tf2_ros REQUIRED)
 find_package(tf2 REQUIRED)
 find_package(diagnostic_updater REQUIRED)
+find_package(OpenCV REQUIRED COMPONENTS core)
```

Comment:
cv-bridge is not enough?
Linux never complain, does it pop up in Win only?

Reply:
Maybe we can add only on WIN32?

Reply:
in Linx jammy/focal, opencv libs are installed by default. We need them to make some functions works, for example CV_ASSERT in our code.. this is the only usage I saw for it. Maybe if we remove this assert we can remove this dependency..